tests/unit/components/tumblr-blog-test.js

```diff
@@ -24,4 +24,8 @@ test('defaults', function (assert) {
   assert.ok(!Ember.isPresent(component.get('postsRoute')), 'postsRoute is not defined by default');
   assert.ok(!component.get('collapsible'), 'blog is not collapsible by default');
   assert.ok(component.get('collapseByDefault'), 'blog is set to collapse by default if collapsible');
+
+  const sortBy = component.get('sortBy');
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).